packages/next/src/client/app-index.tsx

```diff
@@ -229,6 +230,16 @@ function Root({ children }: React.PropsWithChildren<{}>) {
     }, [])
   }
 
+  if (process.env.NODE_ENV !== 'production') {
+    const ssrError = devQueueSsrError()
```

Comment:
This doesn't really work. You're reading a mutable source during render. Why do we need to read these from the DOM instead of passing the data around?

Reply:
I see now. I guess we're never re-rendering the `<template>` and assume it's already there when we render here? Should still move the read into the effect though

Reply:
that would be late and we can't get the error.